api/src/services/listing.service.ts

```diff
@@ -935,15 +936,40 @@ export class ListingService implements OnModuleInit {
       throw new BadRequestException('New listing name must be unique');
     }
 
+    const duplicateListingPermissions = (
+      requestingUser?.jurisdictions?.length === 1
+        ? requestingUser?.jurisdictions[0]
+        : requestingUser?.jurisdictions?.find(
+            (juris) => juris.id === storedListing?.jurisdictions?.id,
+          )
+    )?.duplicateListingPermissions;
+
     const userRoles =
-      process.env.ALLOW_PARTNERS_TO_DUPLICATE_LISTINGS === 'TRUE' &&
-      (requestingUser?.userRoles?.isJurisdictionalAdmin ||
-        requestingUser?.userRoles?.isPartner)
+      requestingUser?.userRoles?.isAdmin ||
+      (requestingUser?.userRoles?.isJurisdictionalAdmin &&
+        duplicateListingPermissions?.includes(
+          UserRoleEnum.jurisdictionAdmin,
+        )) ||
+      (requestingUser?.userRoles?.isPartner &&
+        duplicateListingPermissions?.includes(UserRoleEnum.partner))
         ? {
             ...requestingUser.userRoles,
             isAdmin: true,
           }
-        : requestingUser?.userRoles;
+        : {
+            ...requestingUser?.userRoles,
+            isAdmin: false,
+          };
+    //manually check for juris mismatch since logic above is forcing admin permissioning
+    if (
+      (requestingUser?.userRoles?.isJurisdictionalAdmin ||
+        requestingUser?.userRoles?.isPartner) &&
+      !requestingUser?.jurisdictions?.some(
+        (juris) => juris.id === storedListing.jurisdictionId,
+      )
+    ) {
+      throw new ForbiddenException();
+    }
```

Comment:
@mcgarrye This PR is waiting on #4403 but wanted to flag this section to  you to get your thoughts since its connected to the tests we discussed. From what I can tell with the isAdmin overriding, we can't just rely on the permissionService.can to cover when a jurisdictional admin has the permissions to copy but attempts to do it in the wrong jurisdiction.